test/karma.tests.js

```diff
@@ -1,6 +1,8 @@
 import './utils/performance';
 import './utils/init';
 
+import '@babel/polyfill';
```

Comment:
https://github.com/airbnb/enzyme/issues/1792#issuecomment-416116382

Reply:
Won't that defeat the purpose of running the tests in IE 11?

Reply:
We'll find out once I investigate the 27 failing tests. But in general yes I think it's a big downside. Enzyme doesn't document browser support so they either don't intend for enzyme to be used in the browser or just for evergreen browsers.

Reply:
Sure, it's a tradeoff we can start with. But can we only apply it to IE 11? We apply it all the targets right now 😱.

Reply:
Hopefully. Needs investigation.